python.py

```diff
@@ -0,0 +1,20 @@
+# here is a coment
+
+#
+# block coment
```

Comment:
Typo inside comment in word `coment`. Possible corrections:  `comment` (62), `moment` (33), `comet` (15)